client/www/pages/status.tsx

```diff
@@ -0,0 +1,349 @@
+import Head from 'next/head';
+import {
+  LandingContainer,
+  LandingFooter,
+  MainNav,
+} from '@/components/marketingUi';
+import * as og from '@/lib/og';
+import { useState, useEffect } from 'react';
+import styles from '@/styles/status.module.css';
+
+function getUptimeColor(percentage: number) {
+  const error = '#e5e7eb';
```

Comment:
I would move these outside of the function. You use `e5e7eb` in other places too.